src/components/network-modification-tree-pane.js

```diff
@@ -152,6 +152,8 @@ export const NetworkModificationTreePane = ({
     }, [broadcastChannel]);
 
     const [activeNode, setActiveNode] = useState(null);
+    const activeNodeRef = useRef();
```

Comment:
const activeNodeRef = useRef(activeNode);

Reply:
"activeNodeRef" to remove, not needed anymore

Reply:
"activeNodeRef" to removed